DAO.sol

```diff
@@ -389,6 +389,9 @@ contract DAO is DAOInterface, Token, TokenSale {
             throw;
         }
 
+        if (_debatingPeriod > 8 weeks)
```

Comment:
Oh okay I see, but this is still one of the "do your due diligence" things and check the proposal before voting.

But certainly having a maximum is sensible.